cli/cmds.go

```diff
@@ -227,6 +227,7 @@ func Make() *cobra.Command {
 	pflags.BoolP("check", "c", false, "return an error during diff if formatting is required")
 	pflags.BoolP("verbose", "v", false, "show files as they are processed& additional stats")
 	pflags.BoolP("quiet", "q", false, "quiet mode, only shows block line numbers ")
+	pflags.BoolP("no-color", "C", false, "disable colorful output")
```

Comment:
could we change this to 
```suggestion
	pflags.BoolP("uncolorful", "u", false, "disable colourful output")
```

Reply:
@katbyte Sure, made the change.